src/CalculatorUITestFramework/WindowsDriverServiceBuilder.cs

```diff
@@ -0,0 +1,93 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Appium.Service;
+using OpenQA.Selenium.Appium.Service.Options;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace CalculatorUITestFramework
+{
+    public class WindowsDriverServiceBuilder
```

Comment:
>WindowsDriverServiceBuilder [](start = 17, length = 27)

Since this includes code from appium-dotnet-driver we need to make sure we include the proper third-party-notices in our project (possibly temporarily, if we ask to take this functionality upstream)